<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <class>CheatSearch</class>
 <widget class="QDialog" name="CheatSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cheat Search</string>
  </property>
  <widget class="QTableWidget" name="tableFound">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>351</width>
     <height>501</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="txtSearch">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbValueType">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Int32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Int8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Float</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Double</string>
    </property>
   </item>
    <item>
      <property name="text">
        <string>Int64</string>
      </property>
    </item>
  </widget>
  <widget class="QPushButton" name="btnFirstScan">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>First Scan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNextScan">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Scan</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblCount">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Count: 0</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkHex">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkNot">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>90</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NOT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Value Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbScanType">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Exact</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bigger Than</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Smaller Than</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Value Between</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSearchTo">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTo">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtBaseMemoryAddress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Base Address</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="txtPointerAddress">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pointer Address</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
